src/EnergyPlus/FluidCoolers.cc

```diff
@@ -533,7 +473,7 @@ namespace FluidCoolers {
 				}
 			}
 
-			ErrorsFound = TestFluidCoolerTwoSpeedInputForDesign( cCurrentModuleObject, AlphArray, cNumericFieldNames, cAlphaFieldNames, FluidCoolerNum );
```

Comment:
Sorry, I know it is hard to diff because I moved stuff into the TestFluidCoolerSingleSpeedInputForDesign but that was just so I could call some unit tests just for that. The only line change related to the bug was the line you highlighted. 

The other change of 

ErrorsFound = ErrorsFound || TestFluidCoolerTwoSpeedInputForDesign( cCurrentModuleObje

was something I noticed in the two speed fluid cooler that I added last time. I should not have been overwriting the ErrorsFound variable since it might have had an error condition before that line and the tests in TestFluidCoolerTwoSpeedInputForDesign() might not have errors.